python/lsst/cell_coadds/_single_cell_coadd.py

```diff
@@ -131,6 +136,32 @@ def common(self) -> CommonComponents:
         # Docstring inherited.
         return self._common
 
+    @property
+    def aperture_correction_map(self) -> SingleCellCoaddApCorrMap | None:
```

Comment:
Can we avoid the `None` possibility be allowing the ap corr map to be empty instead?